mobile/src/main/java/org/openhab/habdroid/model/ServerConfiguration.kt

```diff
@@ -62,7 +62,8 @@ data class ServerConfiguration(
     val localPath: ServerPath?,
     val remotePath: ServerPath?,
     val sslClientCert: String?,
-    val defaultSitemap: DefaultSitemap?
+    val defaultSitemap: DefaultSitemap?,
+    val wifiSsid: String?
```

Comment:
We seem to initialize this with lots of "nulls". I would suggest to insert some default null values.

Reply:
I only found two cases where `wifiSsid` is set to `null`: During upgrade from a pre-multi server version and when adding a new server. In latter case every argument except the name is null, so I wouldn't initialize `wifiSsid` with `null` by default.

Reply:
👍 was not sure if it makes sense. Thanks for clarification!